src/components/LoginPanel.tsx

```diff
@@ -8,16 +8,16 @@ import React, { useEffect } from "react";
 import styled, { useTheme } from "styled-components";
 import Title from "@/components/Title/Title";
 import { NavBarHeight } from "@/components/NavBar/NavigationBar";
+import Paper from "@mui/material/Paper";
 
 export const LoginMain = styled.main`
     height: calc(100vh - ${NavBarHeight});
     display: flex;
     align-content: center;
 `;
 
-const MiddleDiv = styled.div`
+const MiddleDiv = styled(Paper)`
```

Comment:
Imo the login panel almost seems slightly too low down? Idk tho

Reply:
I agree